python/ray/train/tests/test_torch_checkpoint.py

```diff
@@ -0,0 +1,19 @@
+import torch
+
+from ray.train.torch import TorchCheckpoint
+
+
+def test_from_state_dict():
+    model = torch.nn.Linear(1, 1)
+    expected_state_dict = model.state_dict()
+    checkpoint = TorchCheckpoint.from_state_dict(expected_state_dict)
+    actual_state_dict = checkpoint.get_model(torch.nn.Linear(1, 1)).state_dict()
+    assert actual_state_dict == expected_state_dict
+
```

Comment:
should we add a test for from_model as well?

Reply:
@justinvyu is on it. See #27056